nhrpd/nhrp_nhs.c

```diff
@@ -296,7 +296,7 @@ static int nhrp_nhs_resolve(struct thread *t)
 {
 	struct nhrp_nhs *nhs = THREAD_ARG(t);
 
-	resolver_resolve(&nhs->dns_resolve, AF_INET, nhs->nbma_fqdn,
+	resolver_resolve(&nhs->dns_resolve, AF_INET, VRF_DEFAULT, nhs->nbma_fqdn,
```

Comment:
not `nhs->ifp->vrf_id`?

Reply:
you may be right. I am not sure however if nhs->ifp is populated systematically or not.